### PR TITLE
new YAML recipe: AppendToSequence

### PR DESCRIPTION
**Pull request by sullis**
this recipe appends a new value to a YAML sequence

### CODE REVIEW

File: rewrite-yaml/src/test/java/org/openrewrite/yaml/AppendToSequenceTest.java [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+import static org.openrewrite.yaml.Assertions.yaml;
+
+class AppendToSequenceTest implements RewriteTest {
+
+    @Test
+    void appendToSequenceHasDashTrue() {
+        rewriteRun(
+          spec -> spec
+                .cycles(1)
```
Comment (sambsnyd) [inline rewrite-yaml/src/test/java/org/openrewrite/yaml/AppendToSequenceTest.java:29]: Why is it necessary for you to customize the number of cycles the test harness runs for?
If this is done correctly it should be stable, making all changes the first time it visits a file and no further changes on any subsequent visit.

Comment (sullis) [reply rewrite-yaml/src/test/java/org/openrewrite/yaml/AppendToSequenceTest.java]: Yes, you are correct.  I fixed an issue in the AppendToSequenceVisitor last night.  This allowed me to delete the cycles(1) customizations.

**Comment by sullis** [comment]
I just added new tests + new Recipe parameters.  Looking for feedback. @sambsnyd  @timtebeek  @jkschneider

**Comment by sambsnyd** [comment]
Thanks @sullis !

**Pull request merged**